src/components/Menu/Menu.style.ts

```diff
@@ -145,6 +145,7 @@ export const MenuItemLink = styled(Link, {
   display: 'flex',
```

Comment:
I strongly recommend to use the Classic [MenuItem](https://mui.com/material-ui/api/menu-item/) instead of a custom one, will avoid many issues in the future starting with this one.

You can set it up as `Link` type via the `component` property